meta.js

```diff
@@ -43,28 +42,83 @@ module.exports = function(values) {
         type: "list",
         name: "cacher",
         message: "Select a cacher solution",
-        choices: [{ name: "Memory", value: "Memory" }, { name: "Redis", value: "Redis" }],
-        when(answers) {
-          return answers.needCacher;
-        },
+        choices: [
+          { name: "Memory", value: "Memory" },
+          { name: "Redis", value: "Redis" }
+        ],
+        when(answers) { return answers.needCacher; },
         default: "Memory"
       },
       {
         type: "confirm",
-        name: "docker",
-        message: "Add Docker files?",
+        name: "dbService",
+        message: "Add DB sample service?",
         default: true
       },
+      {
+        type: "list",
+        name: "database",
+        message: "Select a database solution",
+        choices: [
```

Comment:
It's not in the JS template and not necessary, because it depends on the `MONGO_URI` env var.

Reply:
this is used to build the Connection, instead of having IF and ELSE